src/internal-components/filtering/PopupFilterContainer.tsx

```diff
@@ -28,11 +28,13 @@ const PopoverContainer: React.FunctionComponent<PopoverContainerProps> = props =
 
     return (
         <Popover id="popover-basic">
-            <Popover.Content>
+            <Popover.Body>
                 <p>Show items with value that:</p>
                 <div style={{marginBottom: "1rem"}}>
                     <FilterTypeDropdownButton selectedFilterType={selectedFilterType} filterTypes={props.filterTypes} onChange={filterType => setSelectedFilterType(filterType as FilterType)} showCaret={true}>
-                        {FilterTypeDisplays[selectedFilterType]}
+                        <>
+                            {FilterTypeDisplays[selectedFilterType]}
+                        </>
```

Comment:
No, that's a fragment: https://reactjs.org/docs/fragments.html

It's just a way to package multiple elements as a single element since components can only return one element.

Reply:
In this case it shouldn't be necessary, since the thing in braces here is a string, not a list of elements. I probably did something wrong with types elsewhere -- let me see if I can fix that.